src/components/BookPage/index.js

```diff
@@ -76,6 +77,7 @@ function BookPage() {
          
 
           <p>{blurb}</p>
+          <AddReview id={id}/>
```

Comment:
I should be able to see the new review as soon as I add it.

Reply:
done